apps/site/src/app/(main)/portal/@applicant/VerticalTimeline.tsx

```diff
@@ -50,8 +51,33 @@ function VerticalTimeline({ status }: VerticalTimelineProps) {
 			</li>
 		) : null;
 
-	const rsvp_component =
+	const waiver_component =
 		status === PortalStatus.accepted ? (
+			<li className="flex flex-row items-center border-t">
+				<Image
+					src={BorderCircle}
+					alt="border-circle"
+					width={25}
+					height={25}
+					className="m-6 mr-12"
+				/>
+				Sign waiver
+			</li>
+		) : status === PortalStatus.waived || status === PortalStatus.confirmed ? (
+			<li className="flex flex-row items-center border-t">
+				<Image
+					src={CheckCircle}
+					alt="checked-circle"
+					width={25}
+					height={25}
+					className="m-6 mr-12"
+				/>
+				Waiver signed
+			</li>
+		) : null;
+
+	const rsvp_component =
+		status === PortalStatus.waived ? (
```

Comment:
Thought: sorry for the confusion, I think it might actually help to show this even when waiver hasn't been signed to indicate that there's another step after waiver? cc: @samderanova

Reply:
I think the email we'll be sending out will show that there are other steps, but yes, it wouldn't hurt to show this even when the waiver hasn't been signed yet.